app/models/planning_application.rb

```diff
@@ -488,6 +497,14 @@ def audit_updated!
     end
   end
 
+  def address_or_boundary_geojson_updated?
+    return if updated_address_or_boundary_geojson
+
+    if saved_changes.keys.intersection(ADDRESS_AND_BOUNDARY_GEOJSON_FIELDS).any?
+      update!(updated_address_or_boundary_geojson: true)
+    end
```

Comment:
How about add an audit for this?

Reply:
The updates to any of those fields are already captured in the audit log so hopefully that should cover this